docs/compute/persistent_tpm_and_uefi_state.md

```diff
@@ -45,6 +45,18 @@ spec:
             persistent: true
 ```
 
+
+In order for the persistent tpm volume to be created successfully you must ensure your storage classes and [storage profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) are configured correctly.  
+The persistent tpm volume will be created with the below access mode if one of the constraints for the access mode is true.
+
+* RWX (ReadWriteMany):
+    * the respective storage profile has any claim property set with Filesystem volume mode and RWX access mode.
+    * the kubevirt cluster config has `VMStateStorageClass` set and the storage profile does not exist or the storage profile exists but has no claim property sets.
+
+* RWO (ReadWriteOnce):
+    * the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX.
```

Comment:
nit - set

Reply:
> the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX.

I believe if we change *sets* to *set* it will be incorrect because a [StorageProfile](https://pkg.go.dev/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1#StorageProfileSpec) has claimPropertySets where each claimPropertySet in the claimPropertySets has volume mode Filesystem and RWO mode.

Reply:
I could change this to the below, thoughts?
> the respective storage profile has `claimPropertySets` where all `claimPropertySet` in `claimPropertySets` have Filesystem volume mode and RWO access mode but not RWX.